polkit/pid_start_time_test.go

```diff
@@ -0,0 +1,58 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build linux
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package polkit
+
+import (
+	"os"
+	"syscall"
+	"testing"
+
+	"gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { check.TestingT(t) }
+
+type polkitSuite struct{}
+
+var _ = check.Suite(&polkitSuite{})
+
+func (s *polkitSuite) TestGetStartTime(c *check.C) {
```

Comment:
I wonder if it would make sense to add a test that actually uses mock data for /proc/%d/stat via e.g. a export_test.go that allows to point to a mock-on-disk /proc. The reason is that if we ever refactor getStartTimeForPid() the test for (startTime, ot(Equals), 0) is true for most of the fields in the stat file. So a off-by-one error in the index might escape.